docs-ref-toc/reference-latest.yml

```diff
@@ -1831,7 +1831,7 @@
       landingPageType: Service
     - name: Management
       items:
-      - name: Resource Management - Cognitive Search
```

Comment:
Same as above, update `DisplayName` in this file instead: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv#L135